src/common/telemetry/telemetry-workflow.ts

```diff
@@ -3,20 +3,14 @@ import type {GitHub} from 'projen/lib/github'
 import {NodeProject, NodeProjectOptions} from 'projen/lib/javascript'
 import {runScriptJob} from '../github'
 import {reportTargetAuthToken, telemetryEnableEnvVar} from './collect-telemetry'
-import type {WithTelemetry} from './with-telemetry'
+import type {TelemetryOptions} from './with-telemetry'
 
 /**
  * Options for PullRequestLint
  */
 export interface TelemetryWorkflowOptions
-  extends Partial<Pick<NodeProject, 'runScriptCommand'>>,
```

Comment:
`runScriptCommand` is pulled directly from the project and thus not needed in options.